docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -234,7 +234,7 @@ After configuring the database and connecting to it in Airflow configuration, yo
 
 .. code-block:: bash
 
-    airflow db init
+    airflow dbinit
```

Comment:
This change is wrong in 2.0+, if `airflow db init` did not work for you, it then means you're on a lower version of airflow. What version of airflow are you using?

Reply:
and before 2.0 it was initdb :)